### PR TITLE
Showprofile

### PR DESCRIPTION
**Pull request by Aloxbro**
Made a components for github's profile + styling css for desktop and mobile

Thank you for your mercy

### CODE REVIEW

File: gitvs/src/App.js [hunk 1]
```diff
@@ -1,24 +1,15 @@
...
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <GetProfile name='Alox' 
+      avatar_url='https://avatars3.githubusercontent.com/u/8034782?s=460&v=4' 
+      public_repo='6' 
+      location='Reims'
+      weapons='JavaScript'/>
```
Comment (erabin05) [inline gitvs/src/App.js]: Indentation could be better :
```
<GetProfile 
      name='Alox' 
      avatar_url='https://avatars3.githubusercontent.com/u/8034782?s=460&v=4' 
      public_repo='6' 
      location='Reims'
      weapons='JavaScript'
/>
```

File: gitvs/src/components/GetProfile.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import './getProfile.css'
+import LogoUrlChanger from './LogoUrlChanger'
+
+function GetProfile({name, avatar_url, public_repo, location, weapons}) {
+    return (
+        <div className='card'>
+            <h1 className='githubNickname'>{name}</h1>
+                <div className='avatarBox'>
+                    <img className='avatar' src={avatar_url} alt='avatar'></img>
```
Comment (erabin05) [inline gitvs/src/components/GetProfile.js]: You can use the **name** in **alt** to have a better description

File: gitvs/src/components/LogoUrlChanger.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    case 'Python':
+        urlAffect = 'https://seeklogo.com/images/P/python-logo-A32636CAA3-seeklogo.com.png'
+        break;
+    default:
+        urlAffect = 'https://seeklogo.com/images/W/wrong-way-sign-logo-74101E1446-seeklogo.com.gif'
+}
+    return(                
+    <figure className='logobox'>
+    <img src={urlAffect} alt='langage logo' className='logo'>
+        </img>
```
Comment (erabin05) [inline gitvs/src/components/LogoUrlChanger.js]:
`````` ?

File: gitvs/src/components/LogoUrlChanger.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import './logoUrlChanger.css'
+
+function LogoUrlChanger({urlCheck}){
```
Comment (erabin05) [inline gitvs/src/components/LogoUrlChanger.js]: The name of your component is not quite right :
 - The **component** is a *WeaponLogo* from what I understood.
 - The **switch** is the *logoUrlChanger*. If you had wroten the switch in a function  you could have called this function with that name.

**Review by erabin05 — CHANGES_REQUESTED** [review]
- You can improve indentation, choose of names, and one img tag

File: gitvs/src/App.js (hunk 1 above)
Comment (Aloxbro) [reply gitvs/src/App.js]: Done

File: gitvs/src/components/GetProfile.js (hunk 2 above)
Comment (Aloxbro) [reply gitvs/src/components/GetProfile.js]: Fixed

File: gitvs/src/components/LogoUrlChanger.js (hunk 3 above)
Comment (Aloxbro) [reply gitvs/src/components/LogoUrlChanger.js]: Removed the </img>

File: gitvs/src/components/LogoUrlChanger.js (hunk 4 above)
Comment (Aloxbro) [reply gitvs/src/components/LogoUrlChanger.js]: Component's name is more explicit for its functionality

**Review by erabin05 — APPROVED** [review]
Ok

**Pull request merged**